barretenberg/cpp/src/barretenberg/goblin/goblin.cpp

```diff
@@ -0,0 +1,2 @@
+// NB: This file is here so that goblin_objects will be created
```

Comment:
Good with just removing the WORKTODO here for now. I am itching after the new year to mass split templates into cpp files + instantiations

Reply:
done